CHANGELOG.md

```diff
@@ -1,5 +1,82 @@
 # PowerShell Preview Extension Release History
 
+## v2020.2.0
+### Thursday, February 20, 2020
+
+#### Release of preview work to stable branch
+
+This release, coinciding with the [GA release of PowerShell 7](link to PowerShell 7 release),
+brings a year of work on the PowerShell extension into the stable release.
+The timing of this release is deliberate, since some of the new features
+depend on additions and bugfixes in PowerShell 7,
+and even those that don't generally work better in PowerShell 7
+thanks to many improvements shipping with it.
+
+Some changes that stabilise in this release include:
+
+- Integration of PSReadLine into the Integrated Console
+- Performance and reliability improvements gained
+  by replacing a hand-rolled Language Server Protocol stack
+  with the LSP server library from the Omnisharp project
+- ISE compatibility mode setting
+- Debugging improvements in PowerShell 7
+- End of support for PowerShell v3/v4 and .NET 4.5.2
+- More things...
+
+!! [Note about preview/stable release cadence] !!
+
+You may also notice that the history of the changelog has changed.
+For a full list of changes between this release and the previous stable release,
+see [here](link to Sydney's changelog).

```

Comment:
Where would be a good place for me to post this?

Reply:
In the docs folder maybe?

Reply:
Will open a PR for it later today so that you can link